qulice-checkstyle/src/test/resources/com/qulice/checkstyle/Bar.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Welcome.
+ *
+ * Friend.
+ */
+package foo;
+/**
+ * Just a simple class.
+ * @version $Id $
+ * @author Alex Doe (alex@qulice.com)
+ */
+public class Bar { }
```

Comment:
@prahladyeri same here, please rename the class to a more appropriate name, something that will tell me what is tested here